CONTRIBUTING.md

```diff
@@ -0,0 +1,418 @@
+must fill out the CLA with the same email address that you used to create your
+GitHub account.
+
+Once you are CLA'ed, we'll be able to accept your pull requests. This is
+necessary because you own the copyright to your changes, even after your
+contribution becomes part of this project. So this agreement simply gives us
+permission to use and redistribute your contributions as part of the project.
+
+Note: Your contributions are verified using the email address for which you use
+to sign the CLA. Therefore,
+[setting your GitHub account to private](https://help.github.com/en/articles/setting-your-commit-email-address)
+is unsupported because all commits from private accounts are sent from the
+`noreply` email address.
+
+### Style guide
+
+Knative documentation follows the
+[Google Developer Documentation Style Guide](https://developers.google.com/style/);
+use it as a reference for writing style questions.
+
+## Contributing to the documentation
+
+### Reporting documentation issues
+
+Knative uses Github issues to track documentation issues and requests. If you
+see a problem with the documentation, submit an issue using the following steps:
+
+1. Check the [Knative docs issues list](https://github.com/knative/docs/issues)
+   before creating an issue to avoid making a duplicate.
+2. Use the [correct template](https://github.com/knative/docs/issues/new) for
+   your new issue. There are two templates available:
+   - **Bug report**: If you're reporting an error in the existing documentation,
+     use this template. This could be anything from broken samples to typos.
+     When you create a bug report, include as many details as possible and
+     include suggested fixes to the issue. If you know which Knative component
+     your bug is related to, you can assign the appropriate
+     [Working Group Lead](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md).
+   - **Feature request**: For upcoming changes to the documentation or requests
+     for more information on a particular subject.
+
+Note that code issues should be filed against the
+[individual Knative repositories](http://github.com/knative), while
+documentation issues should go in the
+[`knative/docs`repo](https://github.com/knative/docs/issues). If the issue is
+specific to the [https://knative.dev](https://knative.dev) website, open the
+issue in the [`knative/website`repo](https://github.com/knative/website/issues).
+
+### Working group
+
+The
+[Knative Documentation Working Group](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#documentation)
+meets weekly on Tuesdays and alternates between a 9am PT and a 4:30pm PT time to
+accommodate contributors in both the EMEA and APAC timezones.
+[Click here](https://calendar.google.com/calendar/embed?src=google.com_18un4fuh6rokqf8hmfftm5oqq4%40group.calendar.google.com)
+to see the exact dates on the Knative working group calendar. If you're
```

Comment:
Maybe split this into a new paragraph to make it more readable?

Reply:
Done.